docs/containers-transports.5.md

```diff
@@ -0,0 +1,102 @@
+% CONTAINERS-TRANSPORTS(5) Containers Transports Man Page
+% Valentin Rothberg
+% April 2019
+
+
+# NAME
+containers-transports - description of supported transports for copying and storing container images
+
+
+# DESCRIPTION
+
+### **containers-storage:** [storage-reference]image-id|docker-reference  [@image-id]}
```

Comment:
It's actually `[storage-reference]{image-id|docker-reference[@image-id]}`

The problem was that when using `(` instead of `{`, the man page adds a line break which entirely breaks readability.

Reply:
Ah, OK.  (The pedantic me would like consistent formatting differentiation between literal syntax elements like `:` and `@`, replaceable identifiers like _tag_, and syntax grouping characters like ( | ) .)